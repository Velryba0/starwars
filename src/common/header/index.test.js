import React from 'react'
import { shallow } from 'enzyme'
import Header from './index'

let wrapped = shallow(<Header/>);

describe('src/common/header', () => {
  it('load component', () => {   
    expect(wrapped).toMatchSnapshot();
  });  
});