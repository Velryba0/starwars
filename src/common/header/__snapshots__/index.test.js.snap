// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header should render the Title Component correctly 1`] = `ShallowWrapper {}`;

exports[`Title should render the Title Component correctly 1`] = `ShallowWrapper {}`;

exports[`src/common/header load component 1`] = `ShallowWrapper {}`;

exports[`src/common/header should render the Title Component correctly 1`] = `ShallowWrapper {}`;

exports[`src/common/header/pagination Pass number 1`] = `ShallowWrapper {}`;
